cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -0,0 +1,68 @@
+import io.commercetools.sunrise.cms.CmsIdentifier;
+import io.commercetools.sunrise.cms.contentful.ContentfulCmsService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentfulCmsServiceIT {
+    private static final List<Locale> SUPPORTED_LOCALES = asList(Locale.GERMAN, Locale.US);
+
+    // credentials for contentful demo account
+    private static final String SPACE_ID = "cfexampleapi";
+    private static final String TOKEN = "b4c0n73n7fu1";
+    private ContentfulCmsService contentfulCmsService;
+
+    @Before
+    public void setUp() throws Exception {
+        contentfulCmsService = ContentfulCmsService.of(SPACE_ID, TOKEN);
+    }
+
+    @Test
+    public void whenAskForExistingStringContentThenGet() throws Exception {
+        CmsIdentifier identifier = CmsIdentifier.ofEntryTypeAndKeyAndField("human", "Finn", "description");
+        Optional<String> content = waitAndGet(contentfulCmsService.get(SUPPORTED_LOCALES, identifier));
+
+        assertThat(content).isPresent();
+        assertThat(content.get()).isEqualTo("Fearless adventurer! Defender of pancakes.");
```

Comment:
not that you need or should to change it, but you could shorten 

```
assertThat(content).isPresent();
assertThat(content.get()).isEqualTo("Fearless adventurer! Defender of pancakes.");
```

```
assertThat(content).contains("Fearless adventurer! Defender of pancakes.");
```
http://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/AbstractOptionalAssert.html
from http://joel-costigliola.github.io/assertj/core-8/api/index.html?overview-summary.html